setup-homebrew/main.sh

```diff
@@ -122,6 +123,11 @@ if [[ ! "$GITHUB_REPOSITORY" =~ ^.+/(home|linux)brew-core$ ]]; then
     unset HOMEBREW_NO_INSTALL_FROM_API
 fi
 
+# Configure HOMEBREW_GITHUB_API_TOKEN (needed for attestations)
+if [[ -n "${BREW_GH_API_TOKEN}" ]]; then
+    echo "HOMEBREW_GITHUB_API_TOKEN=${BREW_GH_API_TOKEN}" >> "${GITHUB_ENV}"
```

Comment:
Will this override workflows that already set `HOMEBREW_GITHUB_API_TOKEN`? Maybe want to check if `HOMEBREW_GITHUB_API_TOKEN` is already set.

Reply:
Yeah, I'll tweak that, thanks for catching!

Reply:
PTAL

Reply:
needs to be something like `${HOMEBREW_GITHUB_API_TOKEN-}` to avoid the unbound variable error but LGTM otherwise